distribution/lib/Standard/Base/0.0.0-dev/src/Data/Set.enso

```diff
@@ -0,0 +1,106 @@
+import project.Any.Any
+import project.Data.Array_Proxy.Array_Proxy
+import project.Data.Map.Map
+import project.Data.Numbers.Integer
+import project.Data.Ordering.Comparable
+import project.Data.Ordering.Ordering
+import project.Data.Vector.Vector
+import project.Data.Text.Extensions
+import project.Data.Text.Text
+import project.Errors.Illegal_Argument.Illegal_Argument
+import project.Nothing.Nothing
+import project.Panic.Panic
+
+from project.Data.Boolean import Boolean, True, False
+
+## UNSTABLE
+   An unordered collection of unique values.
+type Set
+    ## PRIVATE
+    Value (underlying_map : Map Any Boolean)
```

Comment:
```suggestion
    Value (underlying_map : Map Any Nothing)
```

I think having a boolean here suggests that it might be false in some cases, even though false is never used. It should be some token object, maybe Nothing, or ideally we should have a Unit type with a single empty constructor, for things like this.